vue-app/src/components/ProjectComponent.vue

```diff
@@ -0,0 +1,274 @@
+    <img class="project-image" :src="project.bannerImageUrl" :alt="project.name">
+    <div class="content">
+      <div class="about">    
+        <h1 
+          class="project-name"
+          :title="project.address"
+          :project-index="project.index"
+        >
+          <a
+            v-if="klerosCurateUrl"
+            :href="klerosCurateUrl"
+            target="_blank"
+            rel="noopener"
+          >{{ project.name }}</a>
+          <span v-else> {{ project.name }} </span>
+        </h1>
+        <p class="tagline">{{ project.tagline }}</p> 
+        <div class="subtitle">
+          <div class="tag">{{ project.category }} tag </div>
+          <div class="team-byline">By <a href="#team"> {{ project.teamName }} team</a></div>
+        </div>
+        <div class="project-section">
+          <h2>📖 About the project</h2>
+          <div class="project-description" v-html="descriptionHtml"></div>
+        </div>
+        <div class="project-section">
+          <h2>🔧 The problem it solves</h2>
+          <div class="project-description">{{ project.problemSpace }}</div>
+        </div>
+        <div class="project-section">
+          <h2>💰 Funding plans</h2>
+          <div class="project-description">{{ project.plans }}</div>
+        </div> 
+        <div class="address-box">
+          <div>
+            <div class="address-label">Recipient address</div>
+            <div class="address">
+              {{ project.address }} 
+            </div>
+          </div>
+          <div class="copy-div">
+            <div class="copy-btn" @click="copyAddress"><img width="16px" src="@/assets/copy.svg"></div>
+            <div class="copy-btn" @click="copyAddress"><img width="16px" src="@/assets/etherscan.svg"></div>
+          </div>
+        </div>
+        <hr />
+        <div v-if="project.teamName" class="team">
+          <h2>Brought to you by {{ project.teamName }}</h2>
+          <div class="project-description">{{ project.teamDescription }}</div>
+        </div>
+      </div>
+    </div>  
+    <div v-if="previewMode" class="link-box">
+      <h2 class="link-title">Check them out</h2>
+      <div v-if="project.githubUrl" class="link-row">
+        <img src="@/assets/GitHub.svg" />
+        <a :href="project.githubUrl">GitHub repo</a>
+      </div>
+      <div v-if="project.twitterUrl" class="link-row">
+        <img src="@/assets/Twitter.svg" />
+        <a :href="project.twitterUrl">@Twitter</a>
+      </div>  
+      <div v-if="project.websiteUrl" class="link-row">
+        <img src="@/assets/Meridians.svg" />
+        <a :href="project.websiteUrl">{{ project.websiteUrl }}</a>
+      </div>  
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import Component from 'vue-class-component'
+import { Prop } from 'vue-property-decorator'
+import { Project } from '@/api/projects'
+import { markdown } from '@/utils/markdown'
+
+@Component export default class ProjectComponent extends Vue {
```

Comment:
Just an opinion but `ProjectProfile` instead of `ProjectComponent`? No need to call it a component 😄

Reply:
Yeah that works, I just didn't want to use `Project` again since we already have `/views/Project.vue`, I'll change it